CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## Fleet 4.48.1 (Apr 05, 2024)
```

Comment:
This should be Apr 08, 2024 since that's when the release will be published.

Reply:
I was still aiming to try and squeeze this our today since it was deemed critical

Reply:
Got it. To release today, we'd likely need to remove the 429 backoff unless Reed is able to complete it today.